docs/config.md

```diff
@@ -16,7 +16,7 @@ setting `$AWS_SSO_CONFIG` in your shell or via the `--config` flag.
 The first time you run `aws-sso` and it detects there is no configuration file,
 it will prompt you for a number of questions to give you a basic configuration.
 Afterwords, you can edit the file and adjust the settings as desired or run
-[aws-sso config](commands.md#config).
+[aws-sso setup config](commands.md#config).
```

Comment:
there is no command `setup config` and no anchor for `#config`

Reply:
Yeah, v2 has changes that is not backwards compatible.  The built in help and _most_ of the existing docs have been updated, but I did miss that one!  Anyways, i'm happy to fix this myself at this point.  Thanks for bringing it to my attention.